objgraph.py

```diff
@@ -29,7 +29,7 @@
 __author__ = "Marius Gedminas (marius@gedmin.as)"
 __copyright__ = "Copyright (c) 2008-2014 Marius Gedminas"
 __license__ = "MIT"
-__version__ = "1.8.2.dev0"
+__version__ = "1.9.0dev0"
```

Comment:
For future reference, here's how this looks:
```
mg@platonas: ~/src/objgraph [git:master $=] $ make test
python tests.py
....................../home/mg/.venv/local/lib/python2.7/site-packages/setuptools/dist.py:283: UserWarning: The version specified requires normalization, consider using '1.9.0.dev0' instead of '1.9.0dev0'.
  self.metadata.version,
...
----------------------------------------------------------------------
Ran 25 tests in 2.619s

OK
```